vpr/src/place/move_utils.cpp

```diff
@@ -8,9 +8,8 @@
 //Run-time flag to control when placer debug information is printed
```

Comment:
First two lines of this comment can probably be deleted. (get rid of #ifdef and second line seems less detailed / accurate than the rest of the comment).